public/static/pathways/her2_pathway.json

```diff
@@ -190,13 +184,13 @@
     "NoEndocrineTherapy": {
       "key": "NoEndocrineTherapy",
       "label": "No Endocrine Therapy",
-      "action": [],
+      "action": {},
```

Comment:
This should be `null` now, not `{}` right? Same with the one a few lines below

Reply:
Updated suggestion: get rid of the "No Endocrine Therapy" node altogether and put something meaningful in the "Endocrine Therapy" node, like a ServiceRequest for 169413002 | Hormone therapy (procedure)

Reply:
I'll populate them with something meaningful but I think this is important:
>With null action the builder thinks they are "observation" type.

They shouldn't be seen as "observation"...

Reply:
Made action required and updated the HER2+ pathway to have actions on all nodes (remove no endocrine therapy and add a service request for endocrine therapy)